data-reconciliation-app/src/auth/Jwt-validation.ts

```diff
@@ -0,0 +1,42 @@
+import * as ccfapp from "@microsoft/ccf-app";
+import { ServiceResult } from "../utils/service-result";
+import msIdProvider from "./jwtIssuer/MS-AAD";
+import testIdProvider from "./jwtIssuer/TestIdP";
+
+/**
+ * JWT Identity Providers
+*/
+export enum JwtIdentityProviderEnum {
+    MS_AAD = "https://login.microsoftonline.com/common/v2.0",
+    Test = "https://demo"
+}
+type identityId = string;
+
+export interface IJwtIdentityProvider {
+    isValidJwtToken(identity: ccfapp.JwtAuthnIdentity): ServiceResult<identityId>
+}
+
+export interface IValidatorService {
```

Comment:
This contract is implemented by some other validators. I wonder if this is better to be on its own `ts` file?